doc/wiki.txt

```diff
@@ -158,7 +158,7 @@ OPTIONS                                                   *wiki-config-options*
     `Windows`:         Standard temporary folder (based on |tempname()|)
 
 *g:wiki_cache_persistent*
-  Specify whether to use persistant caching.
+  Specify whether to use persistent caching.
```

Comment:
I've been caught misspelling this several times now. I should probably learn how to spell...